charts/backbeat-consumer/templates/deployment.yaml

```diff
@@ -45,6 +46,8 @@ spec:
               value: "{{- printf "%s-s3-metadata" .Release.Name | trunc 63 | trimSuffix "-" -}}"
             - name: QUEUE_POPULATOR_DMD_PORT
               value: "9993"
+            - name: MONGODB_HOST
+              value: "{{ .Release.Name }}-mongodb-replicaset-0.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-1.{{ .Release.Name }}-mongodb-replicaset:27017,{{ .Release.Name }}-mongodb-replicaset-2.{{ .Release.Name }}-mongodb-replicaset:27017"
```

Comment:
Should really get SRV-based lookup working. How should we get single-node deployments to work?!

Reply:
Yes for now we mimic what we have in cloudserver,  and we do most tests in replication mode. Could you tackle the mono/multi instance problem in a following dedicated PR which will address the problem globally including cloudserver ?